testing/firefox/fix-rust-target.patch

```diff
@@ -0,0 +1,11 @@
+--- a/build/moz.configure/rust.configure	2017-05-27 10:48:23.927528005 +0000
++++ b/build/moz.configure/rust.configure	2017-05-27 10:52:42.973790729 +0000
+@@ -130,7 +130,7 @@
+             ('arm', 'Linux'): 'armv7-unknown-linux-gnueabihf',
+             ('sparc64', 'Linux'): 'sparc64-unknown-linux-gnu',
+             ('x86', 'Linux'): 'i686-unknown-linux-gnu',
+-            ('x86_64', 'Linux'): 'x86_64-unknown-linux-gnu',
```

Comment:
I extended the patch to cover x86 and armhf; aarch64 is explicitly disabled in the APKBUILD and sparc is unsupported.
Obviously, with no rust, I could not test those.